### PR TITLE
#1945 Added my name to credits page

### LINKED ISSUE #1945: Matthew Tung - credits
### Description

Added my name to credits page

### Acceptance Criteria

seeing my name on credits page

### Proposed Solution

add it to the 'CreditsPage.tsx file'

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by MatthewRTung**
## Changes

Added my name to credits page and ran prettier 

## Screenshots
Screenshot 2024-01-24 at 4 52 13 PM
Screenshot 2024-01-24 at 4 52 49 PM

_If you made UI changes you must post a screenshot of the whole page for each change in 1) a normal sized window and 2) the smallest possible window_

_If you did any manual testing (e.g., with Postman), put screenshots of the http request and before and after of the db_

_If none of this applies, you can delete this section_

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [done] All commits are tagged with the ticket number
- [done] No linting errors / newline at end of file warnings
- [done] All code follows repository-configured prettier formatting
- [done] No merge conflicts
- [done] All checks passing
- [done] Screenshots of UI changes (see Screenshots section)
- [done] Remove any non-applicable sections of this template
- [done] Assign the PR to yourself
- [done] No `yarn.lock` changes (unless dependencies have changed)
- [done] Request reviewers & ping on Slack
- [done] PR is linked to the ticket (fill in the closes line below)

Closes #1945 (issue #1945)

### CODE REVIEW

File: src/frontend/src/pages/CreditsPage/CreditsPage.tsx [hunk 1]
```diff
@@ -124,18 +124,6 @@ const CreditsPage: React.FC = () => {
     { name: 'Aarav Shyamkumar', color: '#FF0000' },
     { name: 'Raghav Mathur', color: '#009933' },
     { name: 'Anika Sharma', color: '#ff0000' },
-    {
```
Comment (RChandler234) [inline src/frontend/src/pages/CreditsPage/CreditsPage.tsx:127]: please don't delete all these people's names

**Review by RChandler234 — CHANGES_REQUESTED** [review]

File: src/frontend/src/pages/CreditsPage/CreditsPage.tsx (hunk 1 above)
Comment (Maffe963) [reply src/frontend/src/pages/CreditsPage/CreditsPage.tsx:127]: Oh shoot I don’t know how that happened

**Review by jessicayzhao — APPROVED** [review]
looks good!

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥

**Pull request merged**